seqr/views/utils/variant_utils.py

```diff
@@ -239,6 +239,12 @@ def get_variant_key(xpos=None, ref=None, alt=None, genomeVersion=None, **kwargs)
     return '{}-{}-{}_{}'.format(xpos, ref, alt, genomeVersion)
 
 
+def _requires_transcript_metadata(variant):
+    if isinstance(variant, list):
+        return _requires_transcript_metadata(variant[0])
+    return variant.get('genomeVersion') != GENOME_VERSION_GRCh38 or variant.get('chrom', '').startswith('M')
```

Comment:
Do you also have to add a case to handle SVs? (I think SVs can be saved variants too)

Reply:
SVs do not have Ensembl transcripts though so they do not need to be checked here